backend/coins/btc/coin.go

```diff
@@ -130,42 +129,8 @@ func (coin *Coin) Unit() string {
 
 // FormatAmount implements coin.Coin.
 func (coin *Coin) FormatAmount(amount int64) string {
-	return strconv.FormatFloat(btcutil.Amount(amount).ToUnit(btcutil.AmountBTC), 'f',
-		-int(btcutil.AmountBTC+8), 64) + " " + coin.Unit()
-}
-
-func formatAsCurrency(amount float64) string {
-	formatted := strconv.FormatFloat(amount, 'f', 2, 64)
-	position := strings.Index(formatted, ".") - 3
-	for position > 0 {
-		formatted = formatted[:position] + "'" + formatted[position:]
-		position = position - 3
-	}
-	return formatted
-}
-
-// FormatAmountAsJSON implements coin.Coin.
-func (coin *Coin) FormatAmountAsJSON(amount int64) coinpkg.FormattedAmount {
 	float := btcutil.Amount(amount).ToUnit(btcutil.AmountBTC)
-	var conversions map[string]string
-	if coin.ratesUpdater != nil {
-		rates := coin.ratesUpdater.Last()
-		if rates != nil {
-			unit := coin.unit
-			if len(unit) == 4 && strings.HasPrefix(unit, "T") {
-				unit = unit[1:]
-			}
-			conversions = map[string]string{}
-			for key, value := range rates[unit] {
-				conversions[key] = formatAsCurrency(float * value)
-			}
-		}
-	}
-	return coinpkg.FormattedAmount{
-		Amount:      strconv.FormatFloat(float, 'f', -int(btcutil.AmountBTC+8), 64),
-		Unit:        coin.Unit(),
-		Conversions: conversions,
-	}
+	return strconv.FormatFloat(float, 'f', -int(btcutil.AmountBTC+8), 64)
```

Comment:
I'd use something like `btc` because `float` is too close to the `float32` and `float64` types.

Reply:
Well, maybe not `btc` because it's the package name. Maybe `a` is best but I know you won't like it.